src/components/Clients/ClientList.jsx

```diff
@@ -35,7 +35,7 @@ const ClientList = () => {
         }}
       >
         <Typography variant="h6" component="h2" mb={2} align="center" color="secondary">
-          Add clients to your list
```

Comment:
Why is this being deleted?

This is what shows in the `Clients` page if the user doesn't have a client list yet.

Reply:
Now I have added the line, by mistake I delete this line and I haven't  see this

Reply:
Think we can fix the formatting of this with the spaces so it wouldn't need to appear for files that needs reviewing